plugins/boxlite-agent-tooling/.agents/lib/subagent.sh

```
@@ -40,6 +43,27 @@
+# The library defines functions and nothing else, so reloading it costs nothing.
+#
+# Loading stays INERT: no variable left in the caller's scope, no output, and no return
+# or exit while sourcing. A library that can abort its own load takes the decision away
+# from callers that source several and report their own failures.
+#
+# The absence is caught where it can be reported instead — `source` on a missing file
+# only warns, and every caller runs under `set -uo pipefail` rather than `-e`, so
+# without a check the gate would continue with no accessor, take the unknown-host
+# branch, and print the full menu: a packaging bug rendered as a plausible instruction.
+# shellcheck source=./hook-host.sh
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/hook-host.sh" 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Not following: ./hook-host.sh was not specified as input (see shellcheck -x).

(SC1091)